### PR TITLE
Minh fix seed

### PR DESCRIPTION
**Pull request by MikeHoang**

### CODE REVIEW

File: app/controllers/admin/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class Admin::MessagesController < Admin::AdminController
+  before_action :get_message, only: [:destroy, :show]
+
+  def index
+    @messages = Message.where(:receiver_id => current_user.id)
```
Comment (lhhieu89) [inline app/controllers/admin/messages_controller.rb]: Have you try to use `Message.received_messages` here?

File: app/controllers/admin/messages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    flash[:success] = "Message created!"
```
Comment (lhhieu89) [inline app/controllers/admin/messages_controller.rb]: Why do we need this message?

File: app/views/partner/promotions/_form.html.slim [hunk 3]
```diff
@@ -1,35 +1,38 @@
...
-			div.field
-				p.col-sm-3.label-title Agent:
-				br
-				= f.text_field :name, class: "form-control"
-		.col-lg-6
-			div.field
-				p.col-sm-3.label-title Address:
-				br
-				= f.text_field :address, class: "form-control"
+		.tab-5
```
Comment (nguyenductoan) [inline app/views/partner/promotions/_form.html.slim]: Remove tab by 2 space character please!

File: app/views/partner/_personal.html.slim [hunk 4]
```diff
@@ -2,7 +2,7 @@
   .col-lg-5.left
     h2.title Trang chủ cá nhân
   .col-lg-7.right
-    input.txtsearch 
+    = render partial: 'search'
```
Comment (nguyenductoan) [inline app/views/partner/_personal.html.slim]: Remove 'partial' please.

File: config/routes.rb [hunk 5]
```diff
@@ -53,7 +54,9 @@
   namespace :partner do
     get '/feed/:id' => 'partner#load_feed', as: 'load_feed'
     resources :promotions
-    resources :students
+    resources :messages
+    resources :students do
```
Comment (nguyenductoan) [inline config/routes.rb]: remove do...end please.

**Pull request merged**